examples/tut_matrix-transpose-local-array.cpp

```diff
@@ -261,17 +268,17 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
       RAJA::statement::Tile<0, RAJA::statement::tile_fixed<TILE_DIM>, RAJA::loop_exec,
         // This statement will initalize local array memory inside a
         // kernel. The cpu_tile_mem policy specifies that memory should be
-        // allocated on the stack. The entries in the RAJA::ParamList
+        // allocated on the stack. The entries in the RAJA::InitList
         // identify RAJA local arrays in the parameter tuple to intialize.
-        RAJA::statement::InitLocalMem<RAJA::cpu_tile_mem, RAJA::ParamList<2>,
+        RAJA::statement::InitLocalMem<RAJA::cpu_tile_mem, RAJA::InitList<0>,
```

Comment:
We're going to have to go through the documentation carefully to make sure the usage of these things is clear.